### PR TITLE
Fix for RADIUSS P&Gs

### LINKED ISSUE #332: Improvements to Radiuss P&G
- Use anchors as suggested [here](https://github.com/LLNL/llnl.github.io/pull/236#pullrequestreview-379874003)
- Update section badges index consistently with the subsection (bui -> bld)
- Add missing links (e.g. references to xsdk)

### PR DESCRIPTION
**Pull request by adrienbernede**
Fixes #332 

- Use anchors to navigate directly to the given Policy/Guideline.
- Update section badges index consistently with the subsection (bui -> bld).
- Add missing links (e.g. references to xsdk now link the the exact content of it).

### CODE REVIEW

File: radiuss/policies/index.md [hunk 1]
```diff
@@ -94,23 +94,23 @@ issue or pull request to help us evolve these policies.
...
+[![Generic badge](https://img.shields.io/badge/M.bld-1-red.svg)](/radiuss/policies/build/#mbld1) &emsp;Package your installation and provide automated builds of dependencies using [Spack](https://spack.io/).
 
-[![Generic badge](https://img.shields.io/badge/R.bld-2-yellow.svg)](/radiuss/policies/build/) &emsp;Compilable projects (e.g. those written in C++ and/or Fortran) must be buildable with [CMake](https://cmake.org).
+[![Generic badge](https://img.shields.io/badge/R.bld-2-yellow.svg)](/radiuss/policies/build/#rbld2) &emsp;Compilable projects (e.g. those written in C++ and/or Fortran) must be buildable with [CMake](https://cmake.org).
 
-[![Generic badge](https://img.shields.io/badge/R.bld-3-yellow.svg)](/radiuss/policies/logging-debugging/) &emsp;Provide build options for production (optimized) and debug deployments.
+[![Generic badge](https://img.shields.io/badge/R.bld-3-yellow.svg)](/radiuss/policies/logging-debugging/rbld3) &emsp;Provide build options for production (optimized) and debug deployments.
```
Comment (hauten) [inline radiuss/policies/index.md]: Need a hash before rbld3?

Comment (hauten) [reply radiuss/policies/index.md]: nm, fixed in a later commit :D

**Comment by hauten** [comment]
Hi @adrienbernede - My crawler says two links are dead (404s). Do these work for you locally?
- /radiuss/policies/tests-ci.md (appears on /policies/all & /policies/documentation)
- /radiuss/policies/logging-debugging/ (appears on /policies/)

**Comment by adrienbernede** [comment]
Hello @hauten,

I found the first one, which may have been there for while.

Couldn’t find the second one though, even `git grep logging` does not give anything relevant.
Are you testing each commit? Because I had this at some point I think.

**Comment by hauten** [comment]
> Couldn’t find the second one though, even `git grep logging` does not give anything relevant.
> Are you testing each commit? Because I had this at some point I think.

Not every commit. The crawler scans the whole site every 5 days and sometimes takes a while to reflect recent changes (due to caching, I guess). So we're good. Thx for checking though.

**Comment by adrienbernede** [comment]
Great!
Could you close the "change request"? It adds "red" in my PR and I can’t fix it :p

**Comment by adrienbernede** [comment]
Thanks 👍

**Pull request merged**